src/components/Plan.jsx

```diff
@@ -133,152 +67,79 @@ const Children = ({ children }) => children;
+          >
+            {plan?.url && (
             <img src={plan.url} alt="" className={classes.plan} />
-          )}
-
-          {spots.map(Spot => {
-            const { Bloqué, Identifiant: spot, x, y, Type, Description, Cumul } = Spot;
-            const [presence, ...rest] = spotPresences[spot] || [];
-
-            const isLocked = Boolean(Bloqué);
-            const isConflict = Boolean(rest.length);
-            const isOccupied = Boolean(presence);
-            const isOwnSpot = Boolean(sameLowC(presence?.tri, tri));
-            const isCumulative = Boolean(Cumul);
-
-            const canClick = Boolean(!isLocked && (!isOccupied || isOwnSpot));
-
-            const tooltip = <SpotDescription md={Description} spot={Spot} />;
-
-            return (
-              <CustomTooltip
-                key={spot}
-                title={(!edit && !isPast) ? tooltip : ''}
-                placement="right"
-                enterDelay={500}
-              >
-                <Fab
-                  className={clsx({
-                    [classes.spot]: true,
-                    [classes.conflict]: isConflict,
-                    [classes.occupied]: isOccupied && !isOwnSpot,
-                    [classes.ownSpot]: isOwnSpot,
-                    [classes.locked]: isLocked,
-                    [`hl-${presence?.tri}`]: presence?.tri,
-                  })}
-                  disabled={isPast}
-                  component={canClick ? 'div' : 'button'}
-                  style={{
-                    left: `${x}px`,
-                    top: `${y}px`,
-                    borderColor: Type?.color?.replace('-', ''),
-                  }}
-                  size="small"
-                  draggable={Boolean(edit)}
-                  onMouseDown={edit && handleMouseDown(Spot)}
-                  onDragEnd={edit && handleDragEnd}
-                  onClick={() => {
-                    if (edit) { return null; }
-
-                    if (!isOccupied && !isLocked) {
-                      const [firstId, ...extraneous] = dayPresences
-                        ?.filter(({ tri: t }) => sameLowC(t, tri)) // Keep only own points
-                        ?.filter(({ spot: s }) => !isCumulativeSpot(s)) // Keep only non cumulative
-                        ?.filter(() => !isCumulative)
-                        ?.map(({ id }) => id);
-
-                      setPresence({ id: firstId, day, tri, spot, plan: place });
-                      extraneous.forEach(i => deletePresence({ id: i }));
-                    }
-
-                    if (isOwnSpot) {
-                      return deletePresence(presence);
-                    }
-
-                    return null;
-                  }}
-                >
-                  {(!edit && presence?.tri) || spot}
-                </Fab>
-              </CustomTooltip>
-            );
-          })}
-        </Box>
-      </DragComponent>
-    </DragWrapper>
+            )}
+
+            {spots.map(Spot => (
```

Comment:
`Spot` should be lowercase, capitalized variables should be reserved for components